templates/slim/section.html.slim

```diff
@@ -13,8 +13,10 @@
 - data_background_image, data_background_size, data_background_repeat,
   data_background_transition = nil
 
+/ find all images under current node with attribute background or canvas
+/ as long as they are of the same level as we are currently processing
 - slide_images = find_by(:context => :image) { |image|
--   image.attr(1) == 'background' || image.attr(1) == 'canvas'
+-   (image.attr(1) == 'background' || image.attr(1) == 'canvas') && image.level == level
```

Comment:
Since I can't re-open a PR, I feel kind of lazy and would wait to have an actual problem (and bug report / test case) because I'm going to sleep and I'll forget (since it's in the closed queue)...